syncd/NotificationQueue.h

```diff
@@ -2,6 +2,7 @@
 
 extern "C" {
 #include <sai.h>
+#include<saimetadata.h>
```

Comment:
why sudennly this is needed ? also space after include

Reply:
All SAI_SWITCH_NOTIFICATION macros are defined in saimetadata.h. Earlier these were defined in lib/sairediscommon.h